src/app/public/modules/adapter-service/adapter.service.ts

```diff
@@ -135,10 +148,27 @@ export class SkyCoreAdapterService {
    * Returns an array of all focusable children of provided `element`.
    *
    * @param element - The HTMLElement to search within.
+   * @param ignoreTabIndex - By default, this function will filter out elements with
+   * a `tabIndex` of `-1`. Setting `ignoreTabIndex = true` will ignore this filter.
+   * @param ignoreVisibility - By default, this function will only return visible elements.
+   * Setting `ignoreVisibility = true` will ignore this filter.
    */
-  public getFocusableChildren(element: HTMLElement): HTMLElement[] {
-    const elements: Array<HTMLElement>
-      = Array.prototype.slice.call(element.querySelectorAll(SKY_TABBABLE_SELECTOR));
+  public getFocusableChildren(
+    element: HTMLElement,
+    ignoreTabIndex: boolean = false,
```

Comment:
I would make both of these arguments separate properties in a complex "settings" object so that the API is easy to read.

```
this.adapterService.getFocusableChildren(element, {
  ignoreTabIndex: true,
  ignoreVisibility: true
});
```

Reply:
Good idea. This has been done.